nanostack-event-loop/eventOS_event.h

```diff
@@ -79,6 +79,16 @@ extern int8_t eventOS_event_handler_create(void (*handler_func_ptr)(arm_event_s
  * */
 extern uint16_t event_core_dynamic_event_peak_get(void);
 
+/**
+ * \brief Event list queue size read
+ *
+ * Function will return current event list size
+ *
+ * \return event queue size
+ *
+ * */
+extern uint16_t event_core_list_size_get(void);
```

Comment:
Why not call this event_queue_size?

Reply:
Kevin mention that that functions is not best for performance. We must go trough list from start to end. and with this list we must be disable interrupt if use ns_list_count